frontend/src/components/app/app.tsx

```diff
@@ -1,79 +1,29 @@
-import { useLocation } from 'hooks/hooks';
+import React from 'react';
 import { AppRoute } from 'common/enums/enums';
-import Counter from 'components/counter/counter';
 import SignIn from 'components/sign-in/sign-in';
 import SignUp from 'components/sign-up/sign-up';
 import { Link, Route, Switch } from 'components/common/common';
-import logo from 'assets/img/logo.svg';
 
 const App: React.FC = () => {
-  const { pathname } = useLocation();
 
   return (
     <>
       <div className="App">
         <div>
           <ul className="App-navigation-list">
-            <li>
-              <Link to={AppRoute.ROOT}>Root</Link>
-            </li>
             <li>
               <Link to={AppRoute.SIGN_IN}>Sign in</Link>
             </li>
             <li>
               <Link to={AppRoute.SIGN_UP}>Sign up</Link>
             </li>
           </ul>
-          <p>Current path: {pathname}</p>
         </div>
         <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
           <Switch>
-            <Route path={AppRoute.ROOT} component={Counter} exact />
             <Route path={AppRoute.SIGN_IN} component={SignIn} exact />
             <Route path={AppRoute.SIGN_UP} component={SignUp} exact />
```

Comment:
I think we need to make the App component absolutely clean, even remove those links. Later we will add real routes here

Reply:
Thanks, I removed everything from the component app.tsx